route/measure.go

```diff
@@ -93,6 +97,19 @@ func Indexed(m ByPoint, points []Point) ByIndex {
 	return measure.Indexed(m, points)
 }
 
+// DependentIndexed is a measure uses a custom cost func to calculate parameter
```

Comment:
Typo here. I think the same text is used at various places.

Reply:
fixed